server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -81,6 +87,7 @@ export function create(
             singleUseTokenCache,
         }),
         throttle(throttler, winston, commonThrottleOptions),
+        throttle(throttler, winston, commonThrottleOptionsCreateDoc),
```

Comment:
@znewton I think we also need to create another throttler instance here. If we use the same restCallThrottler instance, then the limits for per tenant and per cluster would be the same.

Reply:
Introduced a new Throttler instance